packages/e2e/tests/cards/avs/avs.partial.test.js

```diff
@@ -0,0 +1,41 @@
+import { CARDS_URL } from '../../pages';
+import { start } from '../../utils/commonUtils';
+import { REGULAR_TEST_CARD } from '../utils/constants';
+import CardComponentPage from '../../_models/CardComponent.page';
+import { CLIENTSCRIPT_PARTIAL_AVS_WITH_COUNTRY, CLIENTSCRIPT_PARTIAL_AVS_WITHOUT_COUNTRY } from './avs.partial.clientScripts';
+
+const TEST_SPEED = 1;
+const INVALID_POSTALCODE = 'aaaaaaaaaa';
+
+let cardPage = null;
+
+fixture`Card with Partial AVS`.page(CARDS_URL).beforeEach(() => {
+    cardPage = new CardComponentPage();
+});
+
+test('should validate Postal Code if property data.billingAddress.country is provided', async t => {
+    await start(t, 2000, TEST_SPEED);
+
+    await cardPage.cardUtils.fillCardNumber(t, REGULAR_TEST_CARD);
+    await cardPage.cardUtils.fillDateAndCVC(t);
+
+    await t.typeText(cardPage.postalCodeInput, INVALID_POSTALCODE);
+    await t.click(cardPage.payButton);
+
+    await t.expect(cardPage.postalCodeErrorText.innerText).contains('Invalid format. Expected format: 99999999');
+}).clientScripts({ content: CLIENTSCRIPT_PARTIAL_AVS_WITH_COUNTRY });
+
+test('should not validate Postal Code if property data.billingAddress.country is not provided', async t => {
+    await t.setNativeDialogHandler(() => true);
+    await start(t, 2000, TEST_SPEED);
+
+    await cardPage.cardUtils.fillCardNumber(t, REGULAR_TEST_CARD);
+    await cardPage.cardUtils.fillDateAndCVC(t);
+
+    await t.typeText(cardPage.postalCodeInput, INVALID_POSTALCODE);
+    await t.click(cardPage.payButton);
+
+    // Check the value of the alert text
+    const history = await t.getNativeDialogHistory();
```

Comment:
I'm not sure if this a good default?
Shouldn't we maybe throw and error that is not configured or default to some value?

Reply:
I didn't really understand what you meant by 'good default'. Can you elaborate?

If the merchant didn't configure the partial billing address validation by setting the country, then the UI will accept anything.

Reply:
I think we should throw some kind of error. Otherwise most merchants won't notice there's validation since it's not enabled by default.

Reply:
Starting throwing an error would actually break things. We can't throw an error if we don't know the country, and if we assume that the country would be `US` by default (as partial address is mainly used there), it would break integrations done by brazilian merchants for example.

There is a ticket for the docs team to document this and explain how the validation is be done with partial address mode, to avoid confusion.